tools/WinObjC.Language/WinObjC.Language.Packaging.targets

```diff
@@ -3,11 +3,12 @@
 
   <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents" Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true'">
       <ItemGroup>
-          <!-- Inlcude *all* of /include/  and /include_next/ folder.  -->
-          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\**\*.*"/>
+          <!-- Include *all* of /include/, /include/xplat/ and /include_next/ folder.  -->
+          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\**\*.*" Exclude="$(MSBuildThisFileDirectory)..\include\xplat\**\*.*"/>
+          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\xplat\**\*.*"/>
```

Comment:
Would it be cleaner to not to do this and leave the headers in the package as include\xplat\starboard\*.h?

this way there is consistency in how we include the headers.
So, consumers *in* the tool package use include\xplat to use the headers.
And, consumers *of( the tool package also use include\xplat.

Or something like that.